### PR TITLE
SDK implementation for Radiology Insights (in public preview)

### PR DESCRIPTION
**Pull request by koen-mertens**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by chunyu3 — DISMISSED** [review]
Recording need to be submit to `Azure SDK assets` repo. you can refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.TestFramework/README.md#recording

**Review by jsquire — DISMISSED** [review]
@koen-mertens:

Please make your Azure organization membership public on your GitHub profile.  Looks like you've got the correct repository permissions, but your membership is private.

For information and assistance, please review the Azure SDK onboarding documentation  and use the Onboarding Teams channel:
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

**Comment by koen-mertens** [comment]
> @koen-mertens:
> 
> Please make your Azure organization membership public on your GitHub profile. Looks like you've got the correct repository permissions, but your membership is private.
> 
> For information and assistance, please review the Azure SDK onboarding documentation and use the Onboarding Teams channel:
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

I did change it yesterday. Hope it is adjusted by now…

**Comment by jsquire** [comment]
> > @koen-mertens:
> > Please make your Azure organization membership public on your GitHub profile. Looks like you've got the correct repository permissions, but your membership is private.
> > For information and assistance, please review the Azure SDK onboarding documentation and use the Onboarding Teams channel:
> > 
> > * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> > * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> I did change it yesterday. Hope it is adjusted by now…

Confirmed.  I'm seeing it as public now.   I've kicked the linter job to recheck; should pass now.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -232,7 +232,7 @@
 # ServiceOwners:                                                   @conhua @mengaims @juaduan @moreOver0
 
 # PRLabel: %Cognitive - Health Insights
-/sdk/healthinsights/                                               @asaflevi-ms
+/sdk/healthinsights/                                               @asaflevi-ms @tomsft @koen-mertens
```
Comment (asaflevi-ms) [inline .github/CODEOWNERS:235]: I think you can add your names under radiology insights and not entire health insights SDKs.

**Comment by asaflevi-ms** [comment]
Approved

**Review by asaflevi-ms — APPROVED** [review]

**Pull request merged**